### DISCUSSION TITLE
How do I define a certain portal in command blocks?

### DISCUSSION THREAD

**Discussion by Ookidablobida** [iPortalTeam/ImmersivePortalsMod#539] [Q&A] [answered]
Using /portal cb_’whatever’, how do I specify a portal?

**Comment by janIpen** [comment]
Have you looked in the wiki yet?

**Reply by Ookidablobida** [reply]
Yes, but because the commands are depreciated and are being removed in 1.17, there isn't info on it in there.

**Comment by qouteall** [comment, marked as answer]
https://github.com/qouteall/ImmersivePortalsMod/wiki/Portal-Customization#directly-create-portals
